src/onetime/entrypoints/web/Flask/views.py

```diff
@@ -0,0 +1,47 @@
+
+from flask import request, url_for, render_template, session
+from .app import app
+from .form import SecretCreateForm
```

Comment:
full imports, please.
please use pre-commit hooks, in the hooks, there is a functionality that makes all import to be absolute.

Reply:
solved